source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,75 @@
+
+The rules regarding which departments and document types world taxons apply to are enforced by Whitehall but not by content-tagger meaning that these rules are not enforced uniformly.
+
+We also have a format of [world location][world-location] that was the predecessor to the world taxon pages. This is still used for a number of things ([atom feed][atom-feed], tagging in Whitehall, rendering the [world page][world-page]) so we’ve effectively got two concepts of the same thing and need to always keep them in sync.
+
+Content associated with a world location is tagged to the previous concept of a world location rather than a world taxon. For example https://www.gov.uk/government/news/information-and-events-for-uk-nationals-living-in-portugal has no data link to https://www.gov.uk/world/portugal and instead only has a link to the now defunct Portugal world location page. A web link for the taxon is then [created][create-link] based on the URL.
+
+World taxons are treated the same as topic taxons in content tagger, the only clue a user has that they are dealing with something different is the /world part of their path.
+
+Categories haven't been nurtured or refined so the world taxon structure is dense and repetitive. This results in a poor user experience for anyone tagging to it via Whitehall for example.
+
+
+[tech-debt]: https://trello.com/c/2vlS4GHH/116-we-have-a-rogue-branch-of-the-taxonomy-for-world-locations-that-isnt-part-of-the-official-taxonomy
+[world-location]: https://github.com/alphagov/govuk-content-schemas/blob/0c6097e6afa6c7679b97aa4331b5d1fdd75fcdc3/formats/world_location.jsonnet
+[atom-feed]: https://www.gov.uk/world/yemen.atom
+[world-page]: https://www.gov.uk/world
+[create-link]: https://github.com/alphagov/government-frontend/blob/22c66f4bed2fe2b9eddc3bf76301be8fd3ab9035/app/services/world_location_base_path.rb
+
+## Editing the world taxonomy
+
+Taxons within the world taxonomy can be edited via the same means as [editing topic taxons][edit-taxons].
+
+[edit-taxons]: https://docs.publishing.service.gov.uk/manual/taxonomy.html#editing-the-topic-taxonomy
+
+## Root Taxon
+
+The taxon at the root of the world taxonomy (`/world`) is rendered by whitehall because there currently isn’t a template in collections that is suitable for rendering long lists without descriptions. This means that the actual root taxon as far as content-tagger is concerned is `/world/all` which is redirected to `/world`.
```

Comment:
I wonder if this could be just flagged as one of the problems?

Reply:
done :)